package.json

```diff
@@ -18,16 +49,22 @@
       }
     }
   },
-  "dependencies": {
-    "@octokit/rest": "^20.1.0",
-    "daskeyboard-applet": "^2.11.5",
-    "dotenv": "^16.4.5",
-    "octokit": "^3.2.0"
-  },
-  "devDependencies": {
-    "jest": "^29.7.0"
-  },
-  "scripts":{
-    "test":"jest"
-  }
+  "questions": [
```

Comment:
You have an issue on where questions should be please look at documentation or open source examples